training/templates/principal.html

```diff
@@ -0,0 +1,5 @@
+{% extends 'admin/master.html' %}
+
+{% block body %}
+  <p>Hello world</p>
+{% endblock %}
```

Comment:
missing new line

Reply:
It will be fixed in the next PR in the next branch.